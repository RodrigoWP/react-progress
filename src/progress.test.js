import React from 'react'
import { shallow } from 'enzyme'
import { Progress } from './index'

describe('<Progress />', () => {
  it('should be render correctly', () => {
    const progress = shallow(
      <Progress />
    )

    expect(progress).toMatchSnapshot()
  })

  it('should be set backgroundColor correctly', () => {
    const backgroundColor = '#000'

    const progress = shallow(
      <Progress
        backgroundColor={backgroundColor}
      />
    )

    expect(progress.prop('backgroundColor')).toEqual(backgroundColor);
  })

  it('should be set progressColor correctly', () => {
    const progressColor = '#fff'

    const progress = shallow(
      <Progress
        progressColor={progressColor}
      />
    )

    expect(progress.prop('progressColor')).toEqual(progressColor);
  })

  it('should be set height correctly', () => {
    const height = 10

    const progress = shallow(
      <Progress
        height={height}
      />
    )

    expect(progress.prop('height')).toEqual(height);
  })
})
