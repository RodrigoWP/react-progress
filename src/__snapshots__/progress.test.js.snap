// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Progress /> should be render correctly 1`] = `
<styled.div
  backgroundColor="#d3d3d3"
  height={3}
  progressColor="#6197c6"
>
  <div
    className="indeterminate"
  />
</styled.div>
`;
